### PR TITLE
Integrate/i18n

### PR DESCRIPTION
**Pull request by Mcastres**
PR that implements i18n with your modifications @soupette

### CODE REVIEW

File: client/src/containers/RestaurantsPage/index.js [hunk 1]
```diff
@@ -18,24 +19,26 @@ import getQueryParameters from '../../utils/getQueryParameters';
...
     history.push({
-      search: `?category=${where.category}&district=${where.district}&start=${nextStart}`,
+      search: `?category=${where.category}&district=${where.district}&locale=${where.locale}&start=${nextStart}`,
     });
   };
 
   const getWhereParams = () => {
     const category = getQueryParameters(search, 'category') || 'all';
     const district = getQueryParameters(search, 'district') || '_all';
+    const locale = getQueryParameters(search, 'locale') || 'en';
```
Comment (soupette) [inline client/src/containers/RestaurantsPage/index.js:36]: Do you have a way to access the default locale from the graphQL query?

File: client/src/containers/RestaurantsPage/index.js [hunk 2]
```diff
@@ -134,6 +145,7 @@ RestaurantsPage.propTypes = {
   location: PropTypes.shape({
     search: PropTypes.string.isRequired,
   }),
+  locales: PropTypes.object.isRequired,
```
Comment (soupette) [inline client/src/containers/RestaurantsPage/index.js:148]: This should be an array.

**Review by soupette — CHANGES_REQUESTED** [review]

File: client/src/components/FooterColumn/Column.js [hunk 3]
```diff
@@ -17,11 +16,11 @@ function Column({ links }){
   }, []);
 
   return (
-    <div className="nav-list">
+    <div className="nav-list" key={links.name}>
```
Comment (soupette) [inline client/src/components/FooterColumn/Column.js:19]: This key is not needed. Plus the `links` is an Array

File: client/src/components/FooterColumn/Column.js [hunk 4]
```diff
@@ -17,11 +16,11 @@ function Column({ links }){
...
       {formattedLinks.map(link => {
         if (link.to.startsWith('https')) {
           return (
-            <a href={link.to} target="_blank" rel="noreferrer" key={link.to}>
+            <a href={link.to} target="_blank" rel="noreferrer" key={link.name}>
```
Comment (soupette) [inline client/src/components/FooterColumn/Column.js:23]: Use `link.to` for the key as the `to` should be unique.

**Review by soupette — CHANGES_REQUESTED** [review]

File: client/src/containers/RestaurantsPage/index.js (hunk 1 above)
Comment (Mcastres) [reply client/src/containers/RestaurantsPage/index.js:36]: I don't think we can access the default locale from GraphQL

Comment (Mcastres) [reply client/src/containers/RestaurantsPage/index.js:36]: WDYT @soupette?

Comment (soupette) [reply client/src/containers/RestaurantsPage/index.js:36]: I don't know either to be honest.

**Review by soupette — APPROVED** [review]
LGTM

**Pull request merged**